src/components/tutorials-sidebar/index.tsx

```diff
@@ -0,0 +1,89 @@
+import { Fragment } from 'react'
+import VisuallyHidden from '@reach/visually-hidden'
+import { IconChevronLeft16 } from '@hashicorp/flight-icons/svg-react/chevron-left-16'
+import isAbsoluteUrl from 'lib/is-absolute-url'
+import StandaloneLink from 'components/standalone-link'
+import SkipToMainContent from 'components/sidebar/components/sidebar-nav/skip-to-main-content'
+import { SidebarNavLink } from 'components/sidebar/components/sidebar-nav/sidebar-nav-menu-item'
+import {
+  ListItemProps,
+  SectionListProps,
+  SectionTitleProps,
+  TutorialSidebarProps,
+} from './types'
+import s from './tutorials-sidebar.module.css'
+import { HorizontalRule as DocsHorizontalRule } from 'components/sidebar/components/sidebar-nav/sidebar-nav-menu-item'
+
+function TutorialsSidebar({
+  backToLink,
+  children,
+  ariaLabelledBy,
+}: TutorialSidebarProps) {
+  return (
+    <nav className={s.root} aria-labelledby={ariaLabelledBy}>
+      <div className={s.backToLink}>
+        <StandaloneLink
+          href={backToLink.href}
+          text={backToLink.text}
+          icon={<IconChevronLeft16 />}
+          iconPosition="leading"
+          textSize={200}
+        />
+      </div>
```

Comment:
`StandaloneLink` supports a `className` prop, did you try that already and find that a wrapper `div` worked out better?

Reply:
I tried that initially - the issue I ran into was that on `focus`, we modify `margin` as well, & that conflicted. The wrapper avoids that issue ☺️